packages/aerodrome/src/Aerodrome.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  type TransactionFilter,
+  type SwapActionParams,
+  compressJson,
+} from '@rabbitholegg/questdk'
+import { type Address, zeroAddress } from 'viem'
+import { Chains, CHAIN_TO_TOKENS } from '@rabbitholegg/questdk-plugin-utils'
+import {
+  AERODROME_ROUTER,
+  ETH_FOR_TOKENS_FRAGMENTS,
+  TOKENS_FOR_ETH_FRAGMENTS,
+  TOKENS_FOR_TOKENS_FRAGMENTS,
+  WETH_ADDRESS,
+} from './constants'
+
+export const swap = async (
+  _params: SwapActionParams,
+): Promise<TransactionFilter> => {
+  const { chainId, tokenIn, tokenOut, amountIn, amountOut, recipient } = _params
+
+  return compressJson({
+    chainId,
+    value: tokenIn === zeroAddress ? amountIn : undefined,
+    to: AERODROME_ROUTER,
```

Comment:
We should almost consider a helper functions for this down the line - maybe it takes the router? The rest is pretty bog-standard right?